systemtest/benchtest/main.go

```diff
@@ -289,6 +289,6 @@ func warmupTimeout(ingestRate float64, events uint, epm, agents, cpus int) time.
 	// the apmbench runner can do.
 	factor := math.Max(1, float64(agents)/float64(cpus))
 	timeoutSeconds := math.Max(float64(events)/ingestRate, 1) *
-		float64(agents) * factor
+		float64(agents*2) * factor
```

Comment:
LGTM, just a thought for later, maybe it makes more sense to have a time dependent warmup rather than events dependent warmup, we can do a time dependent warmup with the event rate specified.

Reply:
I totally agree, we discussed this with Silvia yesterday and agreed that a time based warmup would make much more sense. I'll open an improvement issue.

Reply:
Opened #8621